sdk/easm/Azure.Analytics.Defender.Easm/perf/Azure.Analytics.Defender.Easm.Perf.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
If we're not going to create performance tests, let's remove this folder, as it serves no purpose.

Reply:
I tried to remove the folder, but it threw exception Azure.Analytics.Defender.Easm.Perf.csproj when I tried to generate the sdk again. So, I'm leaving it here for now